modules/phase_field/test/tests/feature_volume_vpp_test/tests

```diff
@@ -1,101 +1,125 @@
 [Tests]
-  [./centroid_output_test]
+  [centroid_output_test]
     type = 'CSVDiff'
     input = 'feature_volume_vpp_test.i'
     csvdiff = 'feature_volume_vpp_test_out_features_0000.csv'
 
     issues = "#11395"
-    requirement = "The FeatureVolumeVectorPostprocessor shall output individual centroid locations when requested."
+    requirement = "The FeatureVolumeVectorPostprocessor shall output individual centroid locations "
+                  "when requested."
     design = "/FeatureVolumeVectorPostprocessor.md"
-  [../]
+  []
 
-  [./centroid_output_test_parallel]
+  [centroid_output_test_parallel]
     type = 'CSVDiff'
     input = 'feature_volume_vpp_test.i'
     csvdiff = 'feature_volume_vpp_test_out_features_0000.csv'
 
     issues = "#11395"
-    requirement = "The FeatureVolumeVectorPostprocessor shall output individual centroid locations when requested."
+    requirement = "The FeatureVolumeVectorPostprocessor shall output individual centroid locations "
+                  "when requested."
     design = "/FeatureVolumeVectorPostprocessor.md"
 
     min_parallel = 2
     prereq = 'centroid_output_test'
-  [../]
+  []
```

Comment:
These previous test has `solve = false` and only executed the broken part of the code once, avoiding triggering the bug of the vector redeclaration. I added a proper test below with multiple steps.